src/api/dependency-discovery/package.json

```diff
@@ -22,7 +22,8 @@
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
     "@senecacdot/satellite": "^1.27.0",
-    "query-registry": "2.2.0"
+    "query-registry": "2.2.0",
+    "supertest": "6.1.6"
```

Comment:
What's this supertest dep used for?

Reply:
ESLint insists that I install it as and production dependency :). We are using it in our tests.

Reply:
Ah okay